tests/test_cmdline.py

```diff
@@ -955,24 +956,31 @@ def test_check_json__no_pyenvs_found__falls_back_to_current_env(fake_project):
 @pytest.mark.parametrize(
     "args,imports,dependencies,expected",
     [
+        pytest.param(
+            ["--check-unused"],
+            ["requests"],
+            ["black", "mypy"],
+            [Analysis.success_message(check_undeclared=False, check_unused=True)],
+            id="check_unused_action_on_default_ignored_unused_dep__outputs_nothing",
+        ),
         pytest.param(
             ["--check-unused", "--ignore-unused", "black", "mypy"],
             ["requests"],
             ["black", "mypy"],
             [Analysis.success_message(check_undeclared=False, check_unused=True)],
-            id="check_unused_action_on_ignored_unused_dep__outputs_nothing",
+            id="check_unused_action_on_overriden_ignored_unused_dep__outputs_nothing",
         ),
```

Comment:
Can we use another module name than `black` or `mypy` here (at least one that is _not_ in `DEFAULT_IGNORE_UNUSED`), to verify that we  are indeed obeying the given `--ignore-unused` option here?

Otherwise a potential bug where `--ignore-unused` no longer overrides the default would not be caught by this test...

Reply:
Nice catch!
Addressed in https://github.com/tweag/FawltyDeps/pull/365/commits/66f403e245037e93478b4e85be5ef90b58f390ba.